<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SKeyPassword</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="icon_search">
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../icons/res.qrc">:/icons/search_icon.svg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="openExternalLinks">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="search">
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Поиск</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>414</width>
           <height>357</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="icon_filter">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../icons/res.qrc">:/icons/filter.svg</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="category">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">text-align: center</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="apps_list"/>
      </item>
      <item>
       <widget class="QPushButton" name="add_password_button">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Добавить пароль</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="change_db_button"/>
    <addaction name="add_password_from_menu"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about_program_button"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="help"/>
  </widget>
  <action name="about_program_button">
   <property name="text">
    <string>О приложении</string>
   </property>
  </action>
  <action name="actiondsdsv">
   <property name="text">
    <string>dsdsv</string>
   </property>
  </action>
  <action name="change_db_button">
   <property name="text">
    <string>Загрузить базу данных</string>
   </property>
  </action>
  <action name="add_password_from_menu">
   <property name="text">
    <string>Добавить пароль</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>add_password_from_menu</sender>
   <signal>triggered()</signal>
   <receiver>add_password_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
